### ISSUE TITLE
Attribute Storing All Property Objects ("_lbh15.py")

### ISSUE THREAD

**Issue by lelaus** [newcleo-dev-team/lbh15#60]
create an instance attribute named `__loaded_prop_obj_list` storing all the property objects read from all the provided modules; this way, the available property objects should be acquired only once when the instance is built;

**Comment by lelaus** [comment]
Due to useless information stored within the keys of the `_available_properties_dict` attribute, it should be preferable to convert the dict into a list of properties, thus changing the `_available_properties_dict` attribute into the `_available_properties_list` attribute.

**Comment by lelaus** [comment]
The retrieval of all the available correlations from within the `__change_properties()` method should not be made by calling the `correlations_available()` class method, since the properties have already been loaded.

**Comment by lelaus** [comment]
The `correlations_available()` has been subdivided into two components, the first one loading the property modules, while the second one extracting the available correlations based on the property modules previously loaded. The second component has been encapsulated within the independent private static method called `__extract_available_correlations`, which is called directly within the `__change_properties` method, thus avoiding to load the property modules each time.

**Comment by panDanieleN** [comment]
Merged into master.

No need to update documentation and tests.
